docs/deep-learning/fhe_assistant.md

```diff
@@ -104,11 +104,11 @@ In the example above, the `x` and `y` layers need quantization before being conc
 z = torch.cat([self.quant_concat(x), self.quant_concat(y)])
 ```
 
-## Debugging compilation errors
+## Debugging compilation errors [TBD]
```

Comment:
This part seems redundant to the newly added "Common compilation errors", shall we delete or reframe it?